wagtail/users/views/users.py

```diff
@@ -82,29 +86,14 @@ def get_user_edit_form():
         return UserEditForm
 
 
-def get_users_filter_query(q, model_fields):
-    conditions = Q()
-
-    for term in q.split():
-        if "username" in model_fields:
-            conditions |= Q(username__icontains=term)
-
-        if "first_name" in model_fields:
-            conditions |= Q(first_name__icontains=term)
-
-        if "last_name" in model_fields:
-            conditions |= Q(last_name__icontains=term)
-
-        if "email" in model_fields:
-            conditions |= Q(email__icontains=term)
```

Comment:
If I'm not mistaken, this behaviour of splitting the input is specific to the users views – none of the other listing views do this. I opted to remove it in favour of the standard behaviour in `BaseListingView` for consistency.

Alternatively, we could bring the behaviour over to `BaseListingView`, but this means the complexity of the `OR` clauses will be MxN (with M = the number of "tokens" in the search query, and N = the number of fields in `search_fields`). Without this behaviour, it's only N.